### ISSUE TITLE
Conflicts when multiple notification are on same page

### ISSUE THREAD

**Issue by iammahmudur** [groundhoggwp/holler-box#4]
Hello,

We've created a little plugin ([RCP + Holler Box](https://github.com/josephsellers/rcp-hollerbox)) that connects the content restrictions from Restrict Content Pro with Holler Box which means you can target different levels of subscription or roles on your site with messages, upsells, surveys or anything else that Holler Box supports. 

But we've noticed an issue which has been discussed here:
https://github.com/josephsellers/rcp-hollerbox/issues/1#issuecomment-319182603

Please find the details as given below:
```
To create a Hollerbox notification we need to create a holler box post. But it conflicts with multiple notification on the same page. For example. If I create a Hollerbox for `Everyone` and create another Hollerbox for the `Paying Member` only.  So when `Paying Members` visit the site they will see both notification. But only 1 will be active with the toggle button. So if user close the both notification he/she will able see only 1 notification again. There will be no button to toggle the other one. Please check the  screenshots. 
When both notification showing. 
https://prnt.sc/g0cdlb
After closing both notification. 
https://prnt.sc/g0ce13

The issue is the button is there. but the opacity is 0 as the holler box JS targets elements by id. So it only works for a single element.

It's more problematic when both notification are in same position. In that case both of them stays on top of each other.
```
Please let me know if you need more information regarding this.

Thanks again,

Mahmudur

**Comment by scottopolis** [comment]
Thanks for the report, should be fixed in 72b0791ce5d6c9f9293570da6ccf1e62e23731b9

Can you download the master branch and give it a try?

**Comment by scottopolis** [comment]
Fixed
